agent/worker_test.go

```diff
@@ -437,6 +531,14 @@ func TestWorkerUpdate(t *testing.T) {
 					},
 					Action: api.AssignmentChange_AssignmentActionRemove,
 				},
+				{
```

Comment:
Urgh, sorry, I put this in the wrong place.  Also, I got the number wrong.

This was meant to go on the `Update` test.  On https://github.com/docker/swarmkit/pull/2036/files/56f05eb41a666b9b7aded3520eda7bafbd45360d#diff-c2c8c23394524d00cb459770c94bd797R534, it says that the test case is for removing items that don't exist, but `config-1` (not `config-2` :( sorry) is missing.  Since it removes every other task and secret, I thought maybe removing all the configs would be useful too.